packages/caravan-fees/src/transactionAnalyzer.ts

```diff
@@ -0,0 +1,54 @@
+import { Transaction } from "bitcoinjs-lib-v5";
+import { TransactionAnalysis, UTXO, FeeRate } from "./types";
+import { isRBFSignaled, calculateEffectiveFeeRate } from "./utils";
+
+/**
+ * Analyze a transaction to determine fee bumping options.
+ * @param transaction The transaction to analyze
+ * @param utxos The UTXOs spent in the transaction
```

Comment:
what is the difference between these? this utxo arg is used to calculate effective fee rate. So are these utxos the possible utxos that could be used in the fee bumping?

Reply:
actually the earlier implementation was wrong ... as initially I was taking the caravan/bitcoin transaction class as our ref for which we would fee bump the tx instead of psbts